src/main/java/com/linkedin/replica/wall/commands/impl/AddPostCommand.java

```diff
@@ -28,7 +28,7 @@ public Object execute() throws ParseException {
         WallHandler dbHandler = (WallHandler) this.dbHandler;
 
         // validate that all required arguments that are passed
-        validateArgs(new String[]{"authorId", "type", "companyId", "privacy", "text", "hashtags", "mentions", "likesCount", "images", "videos", "urls", "commentsCount", "shares", "isCompanyPost", "isPrior"});
+        validateArgs(new String[]{"authorId", "type", "companyId", "privacy", "text", "hashtags", "mentions", "likesCount", "images", "videos", "urls", "commentsCount", "shares", "isCompanyPost", "isPrior", "timestamp", "shares", "headLine", "isArticle"});
```

Comment:
Only validate crucial args

Reply:
you mean that we can pass this as nulls if they are not passed? @Nabila63Ahmed